ArticleTemplates/calloutTemplate.html

```diff
@@ -0,0 +1,11 @@
+<details class="article_callout">
+  <summary class="article_callout_heading">
+    <div class="article_callout_title">
```

Comment:
Out of curiosity, is there a reason to prefer a class name like `article_callout_title` to multiple classes, like `article callout title`?

Reply:
These would lead to two opposite outcomes. My version essentially reduces to a boolean AND: `∀x. article(x) ∧ callout(x) ∧ title(x)`, whereas yours reduces to an OR: `∀x. article(x) ∨ callout(x) ∨ title(x)`. So in the latter case, your element would inherit all the styles from either `article`, `callout` or `title`.